polaris-react/src/components/Banner/Banner.scss

```diff
@@ -255,8 +255,12 @@
 
 .Button {
   @include button-base;
-  @include text-style-button;
   @include focus-ring($border-width: 2px);
+  font-size: var(--p-font-size-100);
+  font-weight: var(--p-font-weight-medium);
+  line-height: var(--p-font-line-height-1);
+  text-transform: initial;
+  letter-spacing: initial;
```

Comment:
The primary action Button that uses this class uses the `unstyledButtonFrom` and passes it `action` prop that's being passed to `Banner` as content. I'm hesitant to update the util just so that it uses `Text` because the style of the text component will vary depending on use case.